tier1-icons.html

```diff
@@ -293,7 +293,7 @@
         <g id="quizzing">
```

Comment:
I agree the page itself should be yes, but not the question mark. Leaving it for pass one... we can try a different approach for these edge cases or get a new icon.